src/argilla/client/datasets.py

```diff
@@ -761,7 +761,6 @@ def _prepare_for_training_with_transformers(
                 {
                     "id": ds["id"],
                     "text": ds["text"],
-                    "context": ds_dict["context"],
```

Comment:
Hi @davidberenstein1957! I'd say you also need to remove the `context` references in the unit tests at https://github.com/argilla-io/argilla/blob/29972416cf83869fb664d78f75e150c99f45ed51/tests/client/test_dataset.py#L346-L355 👍🏻